### PR TITLE
Fix 143

### PR DESCRIPTION
**Pull request by mvarewyck**
This PR fixes

- [x] #143 
- [x] https://github.com/inbo/alien-species-portal/issues/119: include option to change response type to either "absolute", "relative" or "cumulative" numbers. I have therefore replaced the function argument boolean 'relative' by character 'response_type'. 
@damianooldoni should we foresee backward compatability?
- [x] Fix duplicate filtering. It was too strict, filtering out all duplicates per year and location (max value per year was 1 per location), while it should be for year/location/key (multiple species introduced per year&location) (#145).
- [x] Add `NEWS.md` (#140)

I have increased version to 2.3.2 (not sure what are the internal guidelines about this)

### CODE REVIEW

**Review by damianooldoni — CHANGES_REQUESTED** [review]
Thanks @mvarewyck for the improvements of the function!
The changes as they are now are breaking code. And it means that we need to upgrade to version 3.0.0. Typically, I would reply that such changes are not worth a major version change. However, this package is not on CRAN and not widely used. So, I could say that we can upgrade to 3.0.0.

However, please consider the following changes:
- make old arguments (`relative`) deprecated. Set internally `response_type` based on the value given to `relative`. Please, use lifecycle::deprecated() for it. Quite self-explanatory example from [camtraptor::read_camtrap_dp()](https://github.com/inbo/camtraptor/blob/main/R/read_camtrap_dp.R#L51-L64) 
- As we go for 3.0.0, it's definitely time to start a `NEWS.md` from now on. Could you please add such file in root folder? Don't worry about older versions. We will start from 3.0.0. You can follow this example from [camtraptor](https://github.com/inbo/camtraptor/blob/main/NEWS.md) which is automatically rendered as this [website](https://inbo.github.io/camtraptor/news/index.html).

**Comment by mvarewyck** [comment]
This PR will fix #140

**Comment by damianooldoni** [comment]
Hi @mvarewyck 👋 
Is it ok if I review this PR tomorrow morning?

**Comment by mvarewyck** [comment]
> Hi @mvarewyck 👋 Is it ok if I review this PR tomorrow morning?

Sure, ready from my side. 
There are still some notes in the R CMD check, but they are related to existing code, so I ignored them.

File: DESCRIPTION [hunk 1]
```diff
@@ -17,7 +17,7 @@ Authors@R: c(
            comment = c(ORCID = "0000-0001-6413-3185")),
     person("Pieter", "Huybrechts", , "pieter.huybrechts@inbo.be", role = "ctb",
            comment = c(ORCID = "0000-0002-6658-6062")),
-    person("Machteld", "Varewyck", , "machteld.varewyck@openanalytics.eu", role = "ctb",
+    person("Machteld", "Varewyck", , "machteld.varewyck@openanalytics.eu", role = "aut",
```
Comment (damianooldoni) [inline DESCRIPTION:20]: I added you as author of the package as a whole.

**Review by damianooldoni — APPROVED** [review]
Very nice work 👍 Thanks for adding value to this package.

I have updated your role within the package: you are an author 🎉 And so I removed the `@author` field in roxygen documentation of the help function. Otherwise I should do the same for all other functions where others have worked on as well. git commits history is enough to know who did what in the rare case I need to ask about the code.

For the rest, I pushed some minor changes. The commit titles should be self explicative.
If you have any question about them, please let me know.

**Pull request merged**